episodes/09-formative-assessment.md

```diff
@@ -42,6 +20,59 @@ exercises: 35
+
+  - **implemented curriculum** are the concepts, relationships and skills that are explicitly covered in your lesson and translated into classroom practices - i.e. the teaching & learning activities,
+  - **hidden curriculum** includes all unintended curricular effects that influence learning but are not explicitly  addressed in the lesson content - e.g. (unofficial) social and cultural norms, behaviours and values that are transferred by instructors to learners, often unconsciously. These are all the additional things that people learn about the topic from *the way* your lesson is taught, rather than from its official content. You can use Instructor Notes (covered in more detail in ["Preparing to Teach"](18-preparing.md) episode) to guide instructors on some of the aspects of the hidden curriculum, e.g. to encourage them to follow certain tried and tested practices.
+  
+The goal of the remaining steps of lesson development is to ensure that the attained curriculum matches the intended curriculum as closely as possible. To do so, you need to develop assessments to ensure progression towards your learning outcomes.
+
+::::::::::::::::::::::::::::::::::::::  callout
+
+## Useful reading
+
+Useful further reading: [Course design: Considerations for Trainers – a Professional Guide](https://f1000research.com/documents/9-1377).
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Assessements
+
+In order to measure progress and evaluate if and what learning occurred - we use various types of assessments:
+
+ - **formative assessments** - used to detect changes in learner performance *during*
+instruction, to provide feedback and insight into the learners' developing mental models of the topic taught and to identify any old or developing misconceptions. 
+- **summative assessments** - used to verify whether learners achieved
+the stated learning objectives after instruction.
+
+Summative assessments sum up what learning has been achieved after 
+training (e.g. via exams). They give valuable
+data about learning attainment by individuals and entire cohorts but are not used to guide further progress. They may not be as suitable for short courses, but may be necessary for those that give marks or certificates of completion. 
+
+Formative assessments are applied throughout a course; they
+can inform decisions about how to modify instruction to better promote learning; they can also inform learners about changes they may need to make to improve their learning.
```

Comment:
It would be good to mention formative assessment as a way to move information from working memory to long-term memory here too, linking to the relevant section of the Instructor Training curriculum https://carpentries.github.io/instructor-training/05-memory/index.html#using-formative-assessment-to-support-memory-consolidation

Reply:
Fixed via 0de9ec6 - check if this is better now @tobyhodges.